kubetest/extract.go

```diff
@@ -146,7 +146,7 @@ func (l extractStrategies) Extract(project, zone string) error {
 		}
 	}
 
-	return os.Chdir("kubernetes")
+	return nil
```

Comment:
this is probably going to break things

Reply:
@krzyzacy, i have taken care of chdir to kubernetes in here https://github.com/kubernetes/test-infra/pull/5144/files#diff-03f99726d1a108efe0a32fe962ca8f8bR343

Reply:
not all the jobs uses `--extract`, some jobs check out k8s repo directly which already in k8s dir.

Reply:
Oh yeah, I will fix it soon

Reply:
Fixed it, by moving `os.Chdir` inside `if o.extract.Enabled()` condition.